portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ACAdmin.testcase

```diff
@@ -89,4 +92,275 @@ definition {
 		}
 	}
 
+	@description = "Bug: LRAC-11339 | Automation ID: LRAC-11364 | Test Summary: A new user created in the admin part can not access that part but the user Test Test can"
+	@priority = "5"
+	test NewUserCanNotAccessAdminPart {
+		var userFirstName = "ac";
+		var userLastName = "ac";
+		var userScreenName = "ac";
+		var userEmailAddress = "ac@liferay.com";
+
+		task ("Get the URL link") {
+			var currentURL = Navigator.getCurrentURL();
+		}
+
+		task ("Navigate to the AC Admin page") {
+			Navigator.openSpecificURL(url = "${currentURL}admin");
+		}
+
+		task ("Go to control panel > Users > Users and Organizations") {
+			ProductMenuHelper.expandPanel(panel = "control panel");
+
+			ProductMenuHelper.expandCategory(category = "Users");
+
+			ProductMenuHelper.gotoPortlet(portlet = "Users and Organizations");
+		}
+
+		task ("Create a new user") {
+			LexiconEntry.gotoAdd();
+
+			Type(
+				locator1 = "UsersAndOrganizationsEditUser#DETAILS_SCREEN_NAME_FIELD",
+				value1 = "${userScreenName}");
+
+			Type(
+				locator1 = "UsersAndOrganizationsEditUser#DETAILS_EMAIL_ADDRESS_FIELD",
+				value1 = "${userEmailAddress}");
+
+			Type(
+				locator1 = "UsersAndOrganizationsEditUser#DETAILS_FIRST_NAME_FIELD",
+				value1 = "${userFirstName}");
+
+			Type(
+				locator1 = "UsersAndOrganizationsEditUser#DETAILS_LAST_NAME_FIELD",
+				value1 = "${userLastName}");
+
+			AssertClick(
+				locator1 = "Button#SUBMIT",
+				value1 = "Save");
+
+			UserNavigator.gotoUser(userScreenName = "${userScreenName}");
+
+			UserNavigator.gotoPassword();
+
+			Type(
+				locator1 = "UsersAndOrganizationsEditUser#PASSWORD_NEW_PASSWORD_FIELD",
+				value1 = "test");
+
+			Type(
+				locator1 = "UsersAndOrganizationsEditUser#PASSWORD_ENTER_AGAIN_FIELD",
+				value1 = "test");
+
+			Button.clickSave();
+		}
+
+		task ("Log out the current user and login with the new user") {
+			ProductMenuHelper.expandPanel(panel = "user");
+
+			Click(locator1 = "ProductMenu#USER_SIGNOUT");
+
+			Navigator.openSpecificURL(url = "${currentURL}admin");
+
+			Type(
+				locator1 = "TextInput#EMAIL_ADDRESS",
+				value1 = "ac@liferay.com");
+
+			Type(
+				locator1 = "TextInput#PASSWORD",
+				value1 = "test");
+
+			Click(
+				locator1 = "Button#SIGN_IN",
+				value1 = "Sign In");
```

Comment:
Hey @manuferreira, I think you could log in using this macro. What do you think?

```
ACLogin.loginAs(
				emailAddress = "ac@liferay.com",
				password = "test");
```

Reply:
I will check it out. Thanks :)